src/js/control-bar/time-display.js

```diff
@@ -36,12 +36,12 @@ CurrentTimeDisplay.prototype.createEl = function(){
 CurrentTimeDisplay.prototype.updateContent = function(){
   // Allows for smooth scrubbing, when player can't keep up.
   let time = (this.player_.scrubbing) ? this.player_.getCache().currentTime : this.player_.currentTime();
-  this.contentEl_.innerHTML = '<span class="vjs-control-text">' + this.localize('Current Time') + '</span> ' + Lib.formatTime(time, this.player_.duration());
+  this.contentEl_.innerHTML = `<span class="vjs-control-text">${this.localize('Current Time')}</span>${Lib.formatTime(time, this.player_.duration())}`;
```

Comment:
The `Lib.formatTime` feels like too much to be doing inside the string now. Should we pull the result of that into a var first?